src/extensions/starter_dashlet/useConnectedProps.ts

```diff
@@ -0,0 +1,30 @@
+
+import { activeGameId } from '../profile_management/selectors';
+
+import { useTranslation } from 'react-i18next';
+import { useSelector, useStore } from 'react-redux';
+import { getSafe } from '../../util/storeHelper';
+import { IConnectedProps } from './types';
+
+const emptyObj = {};
+
+export const useConnectedProps = (): IConnectedProps => {
```

Comment:
Same problem as above. "useXYZ" implies a hook function, a piece of reusable functionality. But this isn't reusable, it's specifically for one component. I don't think moving this away from the Component class it's intended for makes the code better, on the contrary.